services/app-api/serverless.yml

```diff
@@ -82,19 +64,10 @@ provider:
           Resource:
             - ${self:custom.tableArn}
             - ${self:custom.atomicCounterTableArn}
-        - Effect: "Allow" #Warmup permissions
-          Action:
-            - lambda:InvokeFunction
-          Resource:
-            - !Sub arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:${self:service}-${opt:stage, self:provider.stage}-*
```

Comment:
The warmup plugin now generates a least privilege policy for its warmup functions.  Coupled with the idempotency helper plugin, we're good to go.
This is great... our lambda functions shouldn't have these invoke permissions since they don't need them.